tests/factories/cht/generate.js

```diff
@@ -112,19 +177,28 @@ const createDataWithRealNames = ({ healthCenter, user, nbrClinics = 10, nbrPerso
     .map((_, index) => {
       const { clinic, primaryContact } = createClinic(index, healthCenter);
 
-      const additionalPersons = createAdditionalPersons(nbrPersons, clinic);
+      const kids = createAdditionalKid( Math.floor(nbrPersons / 2), clinic);
+      const adults = createAdditionalWoman( Math.floor(nbrPersons / 2), clinic);
+      adults.unshift(primaryContact);
 
-      const allPersons = [primaryContact, ...additionalPersons];
+      const allPersons = [...kids, ...adults];
 
-      return { clinic, persons: allPersons };
+      return { clinic, persons: allPersons, kids, adults };
```

Comment:
Maybe return like this, so no duplication of data in this return. Then where it needs `persons`, we can do the join of both groups. 
```suggestion
      return { clinic, kids, adults };
```